spec/directions/bake_appendix_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeAppendix do
+  let(:doc) do
+    Kitchen::BookDocument.new(document: Nokogiri::XML(<<~HTML
+      <html>
+      <body>
+        <div data-type="chapter">
+          <div data-type="page">
+            <div data-type="document-title">zzzzzzz</div>
+            <section>
+              <div data-type="title">hello</div>
+              <div>world</div>
+            </section>
+          </div>
+        </div>
+      </body>
+      </html>
+    HTML
+    ))
```

Comment:
Is there a way to create a `PageElement` directly (i.e. without creating a `BookDocument` first)?

Reply:
You can use one of the [helper methods here](https://github.com/openstax/kitchen/blob/master/spec/helpers/factory_helpers.rb) or you can add a new `page_element` helper method in this file e.g.

```ruby
def page_element(page_children_html)
  book_containing(html: "<div data-type='page'>#{page_children_html}</div>").pages.first
end
```